docs/source/examples/array-numpy.rst

```diff
@@ -0,0 +1,13 @@
+Creating Dask arrays from NumPy arrays
+======================================
+
+We can create Dask arrays from NumPy arrays using ``da.from_array``.
+
+.. code-block:: python
+
+    >>> import numpy as np
+    >>> import dask.array as da
+    >>> x = np.arange(1000)
+    >>> y = da.from_array(x, chunks=(100))
+    >>> y.mean().compute()
+    499.5
```

Comment:
How about text like the following:

We can create dask arrays from any object that implements NumPy slicing, like a `numpy.ndarray` or on-disk formats like `h5py` or `netCDF` `Dataset` objects.  This is particularly useful with on disk arrays that don't fit in memory but, for simplicity's sake, we show how this works on a NumPy array.  This isn't particularly valuable (the NumPy array already works in memory just fine) but is easy to play with.

My objective here is to avoid naive users changing all of their perfectly fine NumPy code with dask.array code to no benefit.  We run across these cases occasionally, hence my aversion.